Yank/restraints.py

```diff
@@ -1297,13 +1298,12 @@ class Boresch(ReceptorLigandRestraint):
                     K_r/2 * [|r3 - l1| - r_aA0]^2 +
                     + K_thetaA/2 * [angle(r2,r3,l1) - theta_A0]^2 +
                     + K_thetaB/2 * [angle(r3,l1,l2) - theta_B0]^2 +
-                    + K_phiA/2 * [dihedral(r1,r2,r3,l1) - phi_A0]^2 +
-                    + K_phiB/2 * [dihedral(r2,r3,l1,l2) - phi_B0]^2 +
-                    + K_phiC/2 * [dihedral(r3,l1,l2,l3) - phi_C0]^2
+                    + K_phiA/2 * hav[dihedral(r1,r2,r3,l1) - phi_A0] * pi^2 +
```

Comment:
Functions are always written `f(x)`, not `f[x]`, right?

Reply:
Yes, I think that was leftover from the other nested parenthesis from the previous form, I'll fix that on a second pass